<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>communityQuery</class>
 <widget class="QWidget" name="communityQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color:rgba(0,0,0,0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
    background-color:white;
}
</string>
         </property>
         <item>
          <property name="text">
           <string>查询方式</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>社团名称</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>社团类型</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{ background: transparent; }

</string>
         </property>
         <property name="placeholderText">
          <string>请输入查询关键字</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Query">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_join">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
         </property>
         <property name="text">
          <string>申请加入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">QTableView{
    background-color:rgba(0,0,0,0);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,2,1,1">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,3,1,3">
       <item>
        <widget class="QPushButton" name="btnBefore">
         <property name="styleSheet">
          <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="demo.qrc">
           <normaloff>:/demo/before.png</normaloff>:/demo/before.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="styleSheet">
          <string notr="true">QSpinBox{
    background-color:rgba(0,0,0,0);
}</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="styleSheet">
          <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="demo.qrc">
           <normaloff>:/demo/next.png</normaloff>:/demo/next.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSkip">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
       </property>
       <property name="text">
        <string>跳转</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
